helm-chart/splunk-operator/crds/enterprise.splunk.com_clustermasters.yaml

```diff
@@ -988,6 +990,43 @@ spec:
                       type: object
                     type: array
                 type: object
+              clusterManagerRef:
```

Comment:
Thinking more about it, I think this should not be in CommonSplunkSpec but rather in each CRDs spec which use this since ClusterMaster/ClusterManager do not use it or at least make it optional, what do you think?

Reply:
They are now marked as optional since CRDs can have either `*MasterRef` or `*ManagerRef`. But when it comes to splitting them, we decided to avoid such changes in the BiasLang context.  

There are other instances if you think about it. The ClusterManager doesn't need a `ClusterManagerRef` to itself, so doesn't LicenseManager, MonitoringConsole, etc. We also have BundlePush in Standalone, LicenseManager, etc. 

I think as our CRDs become too big we need to optimize their CRDs scope to only include what is needed.

Reply:
Maybe CSPL-1997 can cover some of these scenarios.